mii/client.py

```diff
@@ -34,14 +35,23 @@ def mii_query_handle(deployment_name):
     Returns:
         query_handle: A query handle with a single method `.query(request_dictionary)` using which queries can be sent to the model.
     """
-    task_name, mii_configs = _get_deployment_info(deployment_name)
-    if mii_configs.enable_load_balancing:
-        return MIIClient(task_name, "localhost", mii_configs.port_number)
+    if deployment_name in mii.non_persistent_models:
+        assert deployment_name in mii.non_persistent_models, f"Could not find '{deployment_name}'"
+        inference_pipeline, task = mii.non_persistent_models[deployment_name]
+        return MIINonPersistentClient(task, deployment_name)
+
     else:
```

Comment:
We can remove this `else` and avoid an extra level of indentation since the previous `if` statement returns from the function